eigentrust/src/attestation.rs

```diff
@@ -33,6 +37,11 @@ pub type ECDSAPublicKey = secp256k1::PublicKey;
 pub type ECDSASignature = ecdsa::RecoverableSignature;
 /// Signature represented with field elements
 pub type SignatureFr = Signature<Secp256k1Affine, Scalar, NUM_LIMBS, NUM_BITS, Secp256k1_4_68>;
+/// Attestation represented with field
+pub type AttestationFr = Attestation<Scalar>;
```

Comment:
I think we should either rename the `AttestationFr` as `AttestationScalar`, or rename the `Attestation<Scalar>` as `Attestation<Fr>`.
Personally, I would go for the `AttestationScalar`.
I believe it is better for future reading.